.evergreen/config.yml

```diff
@@ -356,7 +356,7 @@ tasks:
         - func: "run atlas data lake test"
 
     - name: "test-requireApiVersion"
-      tags: ["versioned_api"]
```

Comment:
Ah, I didn't realize Evergreen supported tags. Was this just never used?

Reply:
Yep, we never used this. libmongoc makes heavy use of tags in their config.